crates/services/txpool_v2/src/tests/stability_test.rs

```diff
@@ -0,0 +1,366 @@
+//! This module is basically a fuzzer that throws random dependent transactions into the pool.
+//! It checks that TxPool can't panic and that its graph and state are
+//! correct(not in the unexpected state).
+//! It relies on the `debug_assert` which are present in the code.
```

Comment:
Do we have any plans of running a real fuzzer on this? Perhaps we should consider creating a follow-up to do so.

Reply:
It is our real fuzzer if we want test transaction dependencies graph. With real transactions that are created randomly it is not possible=( (Because UtxoId are random)

Reply:
Alright fair enough :+1:

Reply:
Thanks for creating the issue but I guess we can close it as not planned :)